frontend/src/features/projects/dispose/interfaces.ts

```diff
@@ -204,6 +206,8 @@ export interface IApiProject {
   properties: IProjectProperty[];
   note: string;
   privateNote: string;
+  exemptionRequested?: boolean;
+  exemptionRational?: string;
```

Comment:
This is coming from the server where it is named "exceptionRational" instead of "exceptionRationale" - not a big deal but should be consistent with naming everywhere else.

Reply:
I had to look up the difference lol
https://writingexplained.org/rational-vs-rationale-difference#:~:text=Rationale%20vs.,reasons%20that%20support%20a%20claim.

Reply:
Yeah, I was noticing that too, it is just the API returns exemptionRational but on the frontend I only displayed it with the spelling `rationale` as I think that is the one we want

Reply:
Going to adjust these though